Source/Core/Core/PowerPC/Interpreter/Interpreter_Branch.cpp

```diff
@@ -2,12 +2,12 @@
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
-#include "Core/PowerPC/Interpreter/Interpreter.h"
```

Comment:
there's magic in clang-format that goes "If the header NAME.h matches NAME.cpp put that at the beginning", but that's not useful here. Not sure if it can be tweaked in the config somehow, or an annotation on the line would be less messy